### PR TITLE
Further education admin

### PR DESCRIPTION
**Pull request by ralph-hawkins**
### Context

We've added support for sixth-form and further education students.

### Changes proposed in this pull request

Update places in the service that mention the education setting to include colleges and further education provision

### Guidance to review

Support screens

### CODE REVIEW

File: app/views/support/responsible_bodies/index.html.erb [hunk 1]
```diff
@@ -33,7 +33,7 @@
         <td class="govuk-table__cell">
           <%= pluralize(responsible_body.users_who_have_signed_in_at_least_once, "user") %> signed in
         </td>
-        <td class="govuk-table__cell"><%= responsible_body.who_will_order_devices&.humanize %></td>
+        <td class="govuk-table__cell"><%= responsible_body.who_will_order_devices_label&.humanize %></td>
```
Comment (jamesrose) [inline app/views/support/responsible_bodies/index.html.erb:36]:
```suggestion
        <td class="govuk-table__cell"><%= responsible_body.who_will_order_devices_label%></td>
```

`who_will_order_devices_label` already humanizes the string internally, so we don't need to do it again here.

File: app/views/support/responsible_bodies/show.html.erb [hunk 2]
```diff
@@ -45,29 +45,33 @@
...
-      <h2 class="govuk-heading-l govuk-!-margin-top-9">Centrally managed schools</h2>
-      <p class="govuk-body"><%= @responsible_body.name %>:</p>
-      <ul id="responsible-body-centrally-managed-stats" class="govuk-list govuk-list--bullet">
-        <li>manages ordering for <%= centrally_managing_count_or_all_schools(responsible_body: @responsible_body) %> of its schools</li>
-        <li>has <%= what_to_order_allocation_list(allocations: @virtual_cap_pools) %> available to order right now</li>
-        <li>has ordered <%= what_to_order_state_list(allocations: @virtual_cap_pools) %></li>
-      </ul>
+      <div class="govuk-grid-row">
+          <div class="govuk-grid-column-two-thirds">
+            <h2 class="govuk-heading-l govuk-!-margin-top-9">Centrally managed schools and colleges</h2>
```
Comment (jamesrose) [inline app/views/support/responsible_bodies/show.html.erb:50]: Was this updated for consistency? As far as I'm aware there won't be any FE institutions using virtual caps.

**Review by jamesrose — APPROVED** [review]
A few minor things, but the code looks good to me. I will defer to @fofr for the bulk of this review for content.

File: app/views/support/responsible_bodies/show.html.erb (hunk 2 above)
Comment (ralph-hawkins) [reply app/views/support/responsible_bodies/show.html.erb:50]: My understanding is there are no 'Further education providers' who are part of an RB. But an LA or trust could have the equivalent of a stand-alone sixth-form college as part of the schools they manage.

File: app/views/support/responsible_bodies/index.html.erb (hunk 1 above)
Comment (ralph-hawkins) [reply app/views/support/responsible_bodies/index.html.erb:36]: 👍  thanks

File: app/views/support/responsible_bodies/show.html.erb (hunk 2 above)
Comment (jamesrose) [reply app/views/support/responsible_bodies/show.html.erb:50]: @ralph-hawkins Technically they are all part of an RB. They just have one RB and one school, with devolved ordering and virtual caps turned off.

**Review by jamesrose — APPROVED** [review]

**Review by fofr — APPROVED** [review]

**Pull request merged**